osf-cli/pom.xml

```diff
@@ -128,6 +128,11 @@
             <artifactId>osf-packager</artifactId>
             <version>${project.parent.version}</version>
         </dependency>
+        <dependency>
```

Comment:
I will leave the version alone in my submission, as you will probably be adjusting it appropriately during your refactoring.